1.1.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+//�������� ���������, ��������� �������� ������� x4 + x3 + x2 + x + 1 �� ��� ���������.
+
+float f(float x)
+{
+	float y = x * x;
+	return y * (y + x) + y + x + 1;
+}
+
+int main()
+{
+
+	printf("Do u want an answer to this exaple?  x^4 + x^3 + x^2 + x + 1\n");

```

Comment:
```suggestion
	printf("Do you want an answer to this example?  x^4 + x^3 + x^2 + x + 1\n");
```
Побольше уважения к пользователю твоей программы